Makefile

```diff
@@ -49,6 +49,13 @@ whitespace: ## Run fix-whitespace in check mode
 fix-whitespace: ## Run fix-whitespace in fix mode
 	fix-whitespace --verbose
 
+# local checks
+
+.PHONY: checks
+checks: whitespace style
+	# this should probably be a rule
```

Comment:
Why not do it as a part of this PR though?

Reply:
Fair enough. I don't feel too strong about it. I just can't stand the look of Makefile when I found it about 90% bit-rooted sometime in 2021, I think. I feel like actions are good for setting up the environment (e.g. installing tools) but not very good at running stuff.

Reply:
We have a few examples of that one as well, but this is actually spread between the setup and run actions. In particular, there is a flag in the setup action that I'm confused about because it looks like it's about the run action.

Reply:
I am hoping this won't bitrot because the whole point is that people should use it before pushing to CI, so hopefully any problems will be reported early.

Reply:
Perhaps it's time for a quick-start guide. Which should emphasize this feature because it's designed to make their life easier: they don't have to wait for CI to catch things they can find and fix locally.

Reply:
That's a good idea.